test/parallel_api/algorithm/alg.sorting/alg.binary.search/binary.search/binary_search.pass.cpp

```diff
@@ -19,7 +19,8 @@
 
 #include <iostream>
```

Comment:
Could you please add `#include "pstl_config.h"` into tests? Seems, macros equal 0 without such "include" for now

Reply:
including the `pstl_test_config.h` file and correcting the macro definition seems to have worked.

Reply:
ok. Feel free to merge the patch